packages/core/test/TextureSystem.tests.ts

```diff
@@ -100,7 +103,7 @@ describe('TextureSystem', function ()
             glTexture.samplerType = SAMPLER_TYPES.INT;
             if (renderer.context.webGLVersion === 2)
             {
-                glTexture.internalFormat = renderer.context.gl.RGBA32I;
+                glTexture.internalFormat = renderer.context['gl'].RGBA32I;
```

Comment:
These variable are using manual look up (e.g., `['gl']`) because they are private, right?

Reply:
That is correct!